samples/StaticHtmlComposites/README.md

```diff
@@ -0,0 +1,46 @@
+# Basic sample for using CallComposite and ChatComposite
```

Comment:
nit: our samples will be switching to using only composites shortly, update this to specify using Call and Chat Composites in a plain html application

Reply:
Reworded.